### PR TITLE
fix ajv strict warnings in if/else-tests

### PR DESCRIPTION
**Pull request by Uzlopak**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: test/if-then-else.test.js [hunk 1]
```diff
@@ -391,8 +391,8 @@ t.test('if/else with const integers', (t) => {
   t.plan(2)
 
   const schema = {
-    type: 'integer',
```
Comment (climba03003) [inline test/if-then-else.test.js:394]: `integer` should be a valid `type`. Why change it to `number`?

ttps://json-schema.org/understanding-json-schema/reference/numeric.html#id4

Comment (Uzlopak) [reply test/if-then-else.test.js:394]: test/if-then-else.test.js 2> strict mode: type "number" not allowed by context "integer" at "9f255981-9e81-41ed-8a47-ec359679ccd7#/if" (strictTypes)

Comment (Uzlopak) [reply test/if-then-else.test.js:394]: @climba03003 
Does this answer your question?

Comment (climba03003) [reply test/if-then-else.test.js:394]: It does answer my question, but I would like to hear why it is added like this in the beginning.
This test is added in https://github.com/fastify/fast-json-stringify/pull/533

cc @ivan-tymoshenko

Comment (ivan-tymoshenko) [reply test/if-then-else.test.js:394]: > I would like to hear why it is added like this in the beginning

It made some sense before this change. https://github.com/fastify/fast-json-stringify/pull/533#discussion_r967678324

It's ok to change it now.

**Review by ivan-tymoshenko — APPROVED** [review]

**Review by Eomm — APPROVED** [review]

**Pull request merged**